services/app-web/src/pages/StateSubmission/RateDetails/PackagesWithSharedRates/PackagesWithSharedRates.tsx

```diff
@@ -0,0 +1,191 @@
+                            value: currentSubmissionData.id,
+                        })
+                    }
+                })
+
+            const packagesList = packagesWithUpdatedAt.sort((a, b) =>
+                a['updatedAt'] > b['updatedAt'] ? -1 : 1
+            )
+            setPackageOptions(packagesList)
+        },
+        onError: (error) => {
+            recordJSException(
+                `indexHealthPlanPackagesQuery: Error querying health plan packages. ID: ${parentSubmissionID} Error message: ${error.message}`
+            )
+            return null
+        },
+    })
+
+    return (
+        <FormGroup
+            error={Boolean(
+                showFieldErrors('hasSharedRateCert') &&
+                    showFieldErrors('packagesWithSharedRateCerts')
+            )}
+        >
+            <FieldYesNo
+                className={styles.radioGroup}
+                id={`hasSharedRateCert.${index}.hasSharedRateCert`}
+                name={`${fieldNamePrefix}.hasSharedRateCert`}
+                label="Was this rate certification uploaded to any other submissions?"
+                showError={Boolean(showFieldErrors('hasSharedRateCert'))}
+            />
+
+            {getIn(values, `${fieldNamePrefix}.hasSharedRateCert`) ===
+                'YES' && (
+                <>
+                    <Label
+                        htmlFor={`${fieldNamePrefix}.packagesWithSharedRateCerts`}
+                    >
+                        Please select the submissions that also contain this
+                        rate certification.
+                    </Label>
+                    <Link
+                        aria-label="View all submissions (opens in new window)"
+                        href={'/dashboard'}
+                        variant="external"
+                        target="_blank"
+                    >
+                        View all submissions
+                    </Link>
+                    <PoliteErrorMessage>
+                        {shouldValidate &&
+                            getIn(
+                                errors,
+                                `${fieldNamePrefix}.packagesWithSharedRateCerts`
+                            )}
+                    </PoliteErrorMessage>
+
+                    <PackageSelect
+                        //This key is required here because the combination of react-select, defaultValue, formik and apollo useQuery
+                        // causes issues with the default value when reloading the page
+                        key={`packageOptions-${key}`}
+                        inputId={`${fieldNamePrefix}.packagesWithSharedRateCerts`}
+                        name={`${fieldNamePrefix}.packagesWithSharedRateCerts`}
+                        statePrograms={statePrograms}
+                        initialValues={getIn(
+                            values,
+                            `${fieldNamePrefix}.packagesWithSharedRateCerts`
+                        ).map(
```

Comment:
Regression: After selecting a shared rate from the dropdown, clicking continue, and then clicking back, the selected shared rate is no longer selected in the dropdown. This also occurs when saving a draft.

The selected package still exists in the data, it’s just not being displayed.

Reply:
I'm wondering if we should add the step of continuing to the next section of the form and then back to the Cypress test.

Reply:
made this a controlled Select, which fixed the issue.  moved the tests to cypress